internal/global/internal_logging.go

```diff
@@ -48,6 +48,13 @@ func Info(msg string, keysAndValues ...interface{}) {
 	globalLogger.V(1).Info(msg, keysAndValues...)
 }
 
+// Warn prints messages about potential issues with the API or SDK.
+func Warn(msg string, keysAndValues ...interface{}) {
+	globalLoggerLock.RLock()
+	defer globalLoggerLock.RUnlock()
+	globalLogger.V(0).Info(msg, keysAndValues...)
```

Comment:
This is the default, and minimum, logging level of logr. If this is used how will users disable this but not error logs?

Reply:
Good point.  I wasn't sure how to fit a `Warn` into this.
since Info is level `1` and Error is level `0`, there isn't a good way to add Warn without increasing the level of Info, but unsure if that has additional implications. 

For simplicity I could remove the `Warn` change and just make this an `Info` log. 

Thoughts?

Reply:
> For simplicity I could remove the `Warn` change and just make this an `Info` log.

Sounds good to me :+1: